src/plugins/intel_cpu/src/nodes/mha.cpp

```diff
@@ -1026,7 +1026,7 @@ void MHA::prepareParams() {
 
                 // don't create brgemm kernels for empty tiles
                 if (M_ != 0 && K_ != 0 && N_ != 0) {
-                    if (brg1BaseIdx == -1)
+                    if (brg1BaseIdx == static_cast<size_t>(-1))
```

Comment:
The same here, on L1005:

```cpp
size_t brg1BaseIdx = [-1](std::numeric_limits<size_t>::max());
```

```suggestion
                    if (brg1BaseIdx == std::numeric_limits<size_t>::max())
```

Reply:
Please use `std::numeric_limits<size_t>::max()` here in the same fashion as for `brg0BaseIdx`.

Reply:
Done